src/ui/qgsvectorlayerpropertiesbase.ui

```diff
@@ -128,7 +128,7 @@
            <string>Symbology</string>
           </property>
           <property name="toolTip">
-           <string>Symbology</string>
+           <string>Control features symbology</string>
```

Comment:
```suggestion
           <string>Control feature symbology</string>
```

or "control feature's symbology"

Reply:
What about "The layer's symbology" since not all layers have features and the symbology is not on a feature basis.